include/tester/ssp/Softphone.hpp

```diff
@@ -12,17 +12,16 @@ class Softphone
 {
 public:
     Softphone(const SoftphoneArguments & args):
-        _account(args.id, args.domain, args.secret, std::bind(&inComingCall, *this, std::placeholders::_1)),
-        _call(&_account, std::bind(&onCallState, *this, std::placeholders::_2)) //TODO
+        _account(args.id, args.domain, args.secret, std::bind(&Softphone::onInComingCall, this, std::placeholders::_1))
     {
+        _call = new SSPCall(&_account, boundOnCallState);
```

Comment:
Why use `boundOnCallState` rather than std::bind?

Reply:
done

Reply:
I use this std::function<void(pj::CallInfo ci, const pj::OnCallStateParam &prm)> boundOnCallState =
        std::bind(&Softphone::onCallState, this, std::placeholders::_1, std::placeholders::_2);
because i need to do in twice

Reply:
I understand, but using `std::bind` twice is fine enough.
If it really bothers you, create a private method that returns the output of `std::bind` and use it instead of a member field.
By the way, `boundOnCallState` is being initialized after declaration below, so in my opinion you should remove that entire declaration down there.

Reply:
done